File: src/main/java/com/vjiki/music/dto/CreateMessageRequest.java
package com.vjiki.music.dto;

import java.util.UUID;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CreateMessageRequest {
    private UUID chatId;
    private UUID senderId;
    private String content;
    private UUID replyToId;
    private String messageType = "TEXT";
    private UUID songId;
    private Integer attachmentCount = 0;
}

File: bin/main/com/vjiki/music/dto/CreateMessageRequest.kt
package com.vjiki.music.dto

import java.util.UUID

data class CreateMessageRequest(
    val chatId: UUID,
    val senderId: UUID,
    val content: String? = null,
    val replyToId: UUID? = null,
    val messageType: String = "TEXT",
    val songId: UUID? = null,
    val attachmentCount: Int = 0
)
